src/payment.js

```diff
@@ -23,15 +23,17 @@ export class Payment {
      * @param {string} options.clientId - Mandatory client id
      * @param {string} [options.redirectUri] - Redirect uri
      * @param {string} [options.env=PRE] - Schibsted account environment: `PRE`, `PRO` or `PRO_NO`
+     * @param {string} [options.publisher] - ZUORA publisher
      * @throws {SDKError} - If any of options are invalid
      */
-    constructor({ clientId, redirectUri, env = 'PRE', window = globalWindow() }) {
+    constructor({ clientId, redirectUri, env = 'PRE', publisher, window = globalWindow() }) {
```

Comment:
I dont think that make publisher required param is good approach, as publisher is sth defined by sapi/Zuora it is unkow param for clients using spid

Reply:
it is not required param, unless you use `purchasePromoCodeProductFlowUrl` function

Reply:
so I'd suggest to pass empty string as default

Reply:
why? is empty string proper value, or? You can just not set this value, and if you don't know what should it be you shouldn't. Then you will be not able to generate zuora flow link